### PR TITLE
Fix linting and format errors

### PR DESCRIPTION
**Pull request by jonathanmetzman**
superseeds #182

### CODE REVIEW

File: src/python/crash_analysis/stack_parsing/stack_symbolizer.py [hunk 1]
```diff
@@ -526,7 +526,6 @@ def filter_binary_path(binary_path):
     app_directory = os.path.dirname(app_path)
     binary_filename = os.path.basename(binary_path)
     nfs_directory = environment.get_value('NFS_ROOT')
-    root_directory = environment.get_value('ROOT_DIR')
```
Comment (jonathanmetzman) [inline src/python/crash_analysis/stack_parsing/stack_symbolizer.py]: I didn't think there's a bug by not using this variable so removing.

File: src/python/bot/tasks/commands.py [hunk 2]
```diff
@@ -308,7 +308,6 @@ def process_command(task):
           environment_string = minimize_job.get_environment_string()
           environment_string += '\nORIGINAL_JOB_NAME = %s\n' % job_name
           job_name = minimize_job_override
-          job = minimize_job
```
Comment (jonathanmetzman) [inline src/python/bot/tasks/commands.py]: I didn't think there's a bug by not using this variable so removing.

**Comment by jonathanmetzman** [comment]
Will TBR since previous PR was approved.

**Comment by oliverchang** [comment]
CI is reporting:
```
Step #1: Running: yapf -d src/appengine/handlers/help_redirector.py
Step #1: Running: pylint src/appengine/handlers/upload_testcase.py
Step #1: | Using config file /workspace/.pylintrc
Step #1: | ************* Module workspace.src.appengine.handlers.upload_testcase
Step #1: | W:545, 2: Parameters differ from overridden 'post' method (arguments-differ)
```

**Comment by jonathanmetzman** [comment]
> CI is reporting:
> 
> ```
> Step #1: Running: yapf -d src/appengine/handlers/help_redirector.py
> Step #1: Running: pylint src/appengine/handlers/upload_testcase.py
> Step #1: | Using config file /workspace/.pylintrc
> Step #1: | ************* Module workspace.src.appengine.handlers.upload_testcase
> Step #1: | W:545, 2: Parameters differ from overridden 'post' method (arguments-differ)
> ```
There's some sort of issue with CI or my local setup then. Because `python butler.py lint` doesn't turn this up.

File: src/python/crash_analysis/stack_parsing/stack_symbolizer.py (hunk 1 above)
Comment (Dor1s) [reply src/python/crash_analysis/stack_parsing/stack_symbolizer.py]: sorry accidental click

**Pull request merged**